content/projects/blueocean/roadmap/index.adoc

```diff
@@ -0,0 +1,4 @@
+---
+layout: redirect
+redirect_url: https://www.jenkins.io/projects/blueocean/contribute/index.adoc
```

Comment:
* Can you change www.jenkins.io to jenkins.io? You'll remove one redirection as the website is jenkins.io and not www.jenkins.io
* The redirection link seems weird to me '.adoc', wouldn't it be ''https://jenkins.io/projects/blueocean/contribute/"
```suggestion
redirect_url: https://jenkins.io/projects/blueocean/contribute/
```